ibtk/src/lagrangian/LEInteractor.cpp

```diff
@@ -1311,6 +1311,108 @@ spread_data(const int stencil_sz,
     }
 #endif
 } // spread_data
+
+// For internal use - convert to an enumeration. This may make it into the
+// public interface one day but for backwards compatibility we accept only
+// strings for now.
+enum KernelName
```

Comment:
Not critical since this is internal to this implementation file, but suggest:
```suggestion
enum KernelType
```
which (I think) is consistent with other `enum`s.